WebApiExtended.NETCore.Swashbuckle/packages.config

```diff
@@ -0,0 +1,10 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
this file is redundant. no need for packages config when you are using new style csproj files